src/components/avatar/Avatar.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+    <div class="avatar">
+        <img
+            v-if="src"
+            :class="'avatar-picture '+ imgClasses"
+            :src="src"
+            :alt="alt">
+        <div
+            v-if="!src && !!username"
+            :class="'avatar-text '+ imgClasses + ' ' + generateBgColorForAvatar"
+        >
+            <p :class="'username-first-letters '+ dynamicClass">{{ setText }}</p>
+        </div>
+    </div>
+</template>
+
+<script>
+import config from '../../utils/config'
+
+export default {
+    name: 'BAvatar',
+    props: {
+        src: String,
+        username: String,
```

Comment:
Too specific name, we should use a generic `label` prop

Reply:
I have changed that to **label** instead of **username** just like other components.